cogs/serverinfo.py

```diff
@@ -195,9 +195,12 @@ def find_server(self, msg):
             data.add_field(name="Users", value=userlist)
             data.add_field(name="Bitrate", value=channel.bitrate)
         elif isinstance(channel, discord.TextChannel):
-            pins = await channel.pins()
             data.add_field(name="Members", value="%s"%len(channel.members))
-            data.add_field(name="Pins", value=len(pins), inline=True)
+            try:
+                pins = await channel.pins()
+                data.add_field(name="Pins", value=len(pins), inline=True)
+            except:
```

Comment:
You shouldn't use `except`; it can unintentionally catch errors that you're not expecting.

Reply:
You shouldn't speak; It can unintentionally give away your idiocy

Reply:
Fuck you, deleting my comments. @appu1232 lyric's abusing me

Reply:
Please explicitly catch HTTPException instead. The presence of other exceptions may due to bugs in code, and we don't want to miss them oht by accident.